framework/src/operators/mod.rs

```diff
@@ -123,6 +127,18 @@ pub trait Batch {
         GroupByBatch::new(self, selector, composer)
     }
 
+    /// Appends a complete operator to the end of the pipeline
+    ///
+    /// Use when processing is complete and no further modifications are necessary.
+    /// Any further operators will have no effect on packets that have been through
+    /// the complete operator.
+    fn complete(self) -> CompleteBatch<Self>
```

Comment:
trying to think if there's a more descriptive term for this op. `skip`? `skip_to_send`? `short_circuit`?

Reply:
fwiw `complete()` works for me, and things like `group.map(reject_too_big).complete()` inside a `group_by` make sense when I read it. That's just me though, I'm happy to rename if there's anything close to a consensus.